PrimaryTests.cs

```diff
@@ -0,0 +1,118 @@
+using Allure.Commons;
+using NUnit.Allure.Attributes;
+using NUnit.Allure.Core;
+using OpenQA.Selenium;
+using Training_Center_Task_15.Pages;
+
+namespace Training_Center_Task_15
+{
+    [AllureNUnit]
+    [AllureParentSuite("Root Suite")]
+    public class PrimaryTests
+    {
+        protected string _url = TestBaseInfo.URL;
+
+        protected string _username = TestBaseInfo.MainUserUsername;
+        protected string _email = TestBaseInfo.MainUserEmail;
+        protected string _password = TestBaseInfo.MainUserPassword;
+
+        private IWebDriver _driver;
+        private LoginPage _loginPage;
+        private AccountInformationPage _accountInformationPage;
+        private AccountCreatedPage _accountCreatedPage;
+        private ProductsPage _productsPage;
+        private CartPage _cartPage;
+        private HomePage _homePage;
+        private Navigation _navigation;
+
+        [OneTimeSetUp]
+        public void Init()
+        {
+            SetUpEnvironmentDriver.StartDriver();
+            _driver = SetUpEnvironmentDriver.GetDriver();
+        }
+
+        [SetUp]
+        public void StartTest()
+        {
+            _driver.Navigate().GoToUrl(_url);
+        }
+
+        [Test(Author = "Ivo Igov", Description = "First test")]
+        [Category("Category One")]
+        [AllureSeverity(SeverityLevel.critical)]
+        public void VerifyUserCanCreateAccount()
+        {
+            NewConsumerInfo newConsumerInfo = new NewConsumerInfo();
+
+            _navigation = new Navigation(_driver);
+            _loginPage = _navigation.ClickSignUpLoginLink();
+
+            _loginPage.SignupNewUser(newConsumerInfo.Username, newConsumerInfo.Email);
+            _accountInformationPage = _loginPage.ClickSignupButton();
```

Comment:
do you really need class fields like this? It is better to set local variables

Reply:
Fixed